src/lib.rs

```diff
@@ -477,6 +497,40 @@ impl Criterion {
         self
     }
 
+    /// Benchmarks multiple functions
+    ///
+    /// All functions get the same input and are compared with the other implementations.
+    /// Works similar to `bench`, but with multiple functions.
+    ///
+    /// ``` ignore-test
+    /// fn bench_seq_fib(b: &mut Bencher, i: &u32) {
+    ///     b.iter(|| {
+    ///         seq_fib(test::black_box(i));
```

Comment:
Ok. I don't get it. Do you oppose the usage of `black_box` in general or just in this case? I'm not sure if we need it there or not. I'm mostly tired of benchmarks getting optimized away so I sprinkle them like stardust on all my code when I benchmark.